### PR TITLE
delete inspec-scanner and bulk-send related infrasctructure

### PR DESCRIPTION
**Pull request by sastels**
# Summary | Résumé

We don't use these anymore. Credentials were deactivated months ago during the secret cleanup.

### CODE REVIEW

**Review by ben851 — APPROVED** [review]
I'm making the assumption that these S3 buckets are indeed only used for the out of band bulk send. 

With that in mind, I've verified that the apply works correctly in scratch, and that the the plans across the rest of the folders are not affected.

As noted in my comments, we will need to manually enable the force-destroy of the two s3 buckets before we merge this PR to staging (and production).

File: aws/common/s3.tf [hunk 1]
```diff
@@ -56,61 +56,6 @@ module "csv_bucket_logs" {
...
-  block_public_acls       = true
-  block_public_policy     = true
-  ignore_public_acls      = true
-  restrict_public_buckets = true
-}
-
-module "bulk_send_logs" {
-  source = "github.com/cds-snc/terraform-modules//S3_log_bucket?ref=v5.1.8"
-
-  bucket_name       = "notification-canada-ca-${var.env}-bulk-send-logs"
```
[the remark this answers is not in the document]
Comment (sastels) [reply aws/common/s3.tf:103]: set in #853 as discussed

File: aws/common/s3.tf [hunk 2]
```diff
@@ -56,61 +56,6 @@ module "csv_bucket_logs" {
   }
 }
 
-
-resource "aws_s3_bucket" "bulk_send" {
```
[the remark this answers is not in the document]
Comment (sastels) [reply aws/common/s3.tf:60]: set in #853 as discussed

**Comment by sastels** [comment]
Note that we need #853 to go to production before we can merge this PR into the repo. Will hold off for a day...

**Pull request merged**